src/platform/routes/LandingPage/index.module.scss

```diff
@@ -0,0 +1,46 @@
+@import "~assets/styles/global";
+
+.landingPageContainer {
+  padding: 20px 30px;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  background: url("~assets/img/pink-bg-right.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.pageContainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row-reverse;
+  margin-top: 50px;
+}
+
+.svgImage {
+  svg {
+    width: 650px;
+    height: 500px;
+  }
+}
+
+.contentContainer {
+  p {
+    color: $secondary;
+    font-size: 30px;
+    font-weight: bold;
+    line-height: 2.9rem;
+    width: 70%;
```

Comment:
For sake of simplicity, I think you should use fixed pixels in width, and set max-width: 90% | 95%.

Please consider this for all sentences on all pages ⚠️

Reply:
Done